rllib/examples/quadx_waypoints.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import os
+
+from ray.rllib.utils.test_utils import check_learning_achieved
+from ray.tune.registry import get_trainable_cls
+import gymnasium as gym
+
+parser = argparse.ArgumentParser()
```

Comment:
We have a standard parser in `rllib/utils/test_utils.py` and a run script which we use for examples. This is something I can also do in a cleanup after merge. Your PR, your choice. As soon as this script runs through we are ready to run premerge tests imo.

Reply:
Going to give this a shot! Do these examples run against tests when changes are made in future PRs?